example/lib/pages/list_filters.dart

```diff
@@ -41,6 +47,7 @@ class FilterListScreen extends StatelessWidget {
             if (state is AvailableFiltersStateSucceeded) {
               return TabBarView(
                 children: <Widget>[
+                  const FavoritesPage(),
```

Comment:
Add new field in state, `favouritesFilters` and reuse ListFiltersWidget. Filters must be: CIColorCube, CIColorMonochrome, GPUMonochrome, GlMonochrome, GPULookuptableFilter, GlLookUpTableFilter

Reply:
Changed ListFiltersWidget.